### PR TITLE
update: Added new action hooks on order details sidebar - Shipping Status

### PR DESCRIPTION
**Pull request by alamgircsebd**
This two hooks will help add more section on order details page in sidebar area

### CODE REVIEW

File: includes/Install/Installer.php [hunk 1]
```diff
@@ -350,6 +351,39 @@ public function create_refund_table() {
         dbDelta( $sql );
     }
 
+    /**
```
Comment (nurul-umbhiya) [inline includes/Install/Installer.php:354]: you need to create db tables from pro version, coz shipping tracking will be a pro feature.

File: templates/orders/listing.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
 $order_status   = isset( $_GET['order_status'] ) ? sanitize_key( $_GET['order_status'] ) : 'all';
 $paged          = isset( $_GET['pagenum'] ) ? absint( $_GET['pagenum'] ) : 1;
 $order_date     = isset( $_GET['order_date'] ) ? sanitize_key( $_GET['order_date'] ) : null;
-$order_statuses = apply_filters( 'dokan_bulk_order_statuses', [
+$order_statuses = apply_filters( 'dokan_bulk_order_statuses22', [
```
Comment (nurul-umbhiya) [inline templates/orders/listing.php:10]: why did you added 22?

**Comment by nurul-umbhiya** [comment]
Has dependency with https://github.com/weDevsOfficial/dokan-pro/pull/1132

**Comment by alamgircsebd** [comment]
Yes, this PR have related PR on dokan pro, so do not merge this one for now.

**Comment by alamgircsebd** [comment]
This PR related with shipping tracking status

**Review by nurul-umbhiya — APPROVED** [review]

**Comment by alamgircsebd** [comment]
Warning type field added on dokan admin setting

Screenshot (30)

**Pull request merged**